.eslintrc

```diff
@@ -0,0 +1,73 @@
+{
+  "parser": "babel-eslint",
+  "extends": [
+    "@webex/eslint-config-react",
```

Comment:
Should we extend from Airbnb's instead of this one?

Reply:
can we do that after we come up with our standard eslint configuration

Reply:
I agree with @lalli-flores, we should use the newly agreed upon eslint config instead of using the soon to be deprecated eslint-config.